package.json

```diff
@@ -5,8 +5,8 @@
   "main": "dist/XRay.js",
   "scripts": {
     "prepublish": "babel src -d dist",
-    "start": "webpack-dev-server",
-    "docs": "NODE_ENV=production webpack",
+    "start": "export NODE_ENV=development && webpack-dev-server",
+    "docs": "export NODE_ENV=production && webpack",
```

Comment:
Using `export` might lead to undesired effects for someone working on that package. This code will lead to _globally_ setting `NODE_ENV` to `production` in current bash session.

Reply:
Changes in config (below) are low level version of `-p` CLI flag, instead of setting up `DefinePlugin` and `UglifyJsPlugin` manually, one can update `docs` npm script to look as following:

```bash
NODE_ENV=production webpack -p
```